src/entry.js

```diff
@@ -22,6 +22,8 @@ module.exports = (argv) => {
     process.exit(1);
   }
 
+  require('update-notifier')({ pkg: require('../package.json') }).notify();
```

Comment:
It's not, but it'll only pop it up once a day. Alternatively, it's easy to [opt out](https://github.com/yeoman/update-notifier#user-settings)